MediaAdminBundle/Tests/Functional/Controller/FolderControllerTest.php

```diff
@@ -12,20 +12,66 @@
 class FolderControllerTest extends AbstractControllerTest
 {
     /**
+     * Set up the test
+     */
+    public function setUp()
```

Comment:
Why ? 
this method is called for any test to login user, no ? 
i redefined it to stop connecting users automatically because i do that manually in every method of my test.